types.go

```diff
@@ -538,7 +594,7 @@ func readPLPType(ti *typeInfo, r *tdsBuffer) (res interface{}) {
 }
 
 func writePLPType(w io.Writer, ti typeInfo, buf []byte) (err error) {
-	if err = binary.Write(w, binary.LittleEndian, uint64(len(buf))); err != nil {
+	if err = binary.Write(w, binary.LittleEndian, uint64(UNKNOWN_PLP_LEN)); err != nil {
```

Comment:
It's because it doesn't work when the length is set, check this comment for more info : https://github.com/pekim/tedious/issues/197#issuecomment-58307082